content/en/changelogs/1.0.0-changelog.md

```diff
@@ -1,10 +1,11 @@
 ---
-title: Version 1.0
-changelog_title: Version 1.0.0
+title: Spinnaker Release 1.0.0
 date: 2017-06-05 11:21:34 -0400
 tags:
```

Comment:
I'm not sure the purpose of the tags anymore. The `deprecated` tag that was used is no longer required.
Maybe we should just delete them to remove noise.